auth-api/src/auth_api/resources/org.py

```diff
@@ -322,6 +322,55 @@ def delete(org_id):
         return response, status
 
 
+@cors_preflight('GET,POST,OPTIONS')
+@API.route('/<int:org_id>/affiliationInvitations', methods=['GET', 'OPTIONS'])
+class OrgAffiliationInvitationsRequestAccess(Resource):
+    """Resource for managing affiliation invitations for an org."""
+
+    @staticmethod
+    def _verify_permissions(org_id):
+        if UserService.is_context_user_staff():
+            return
+
+        user = UserService.find_by_jwt_token()
+
+        if not user:
+            raise BusinessException(Error.NOT_AUTHORIZED_TO_PERFORM_THIS_ACTION, None)
+
+        if not UserService.is_user_member_of_org(user=user, org_id=org_id):
+            raise BusinessException(Error.NOT_AUTHORIZED_TO_PERFORM_THIS_ACTION, None)
+
+    @classmethod
+    @TRACER.trace()
+    @cors.crossdomain(origin='*')
+    @_jwt.has_one_of_roles([Role.SYSTEM.value, Role.STAFF_VIEW_ACCOUNTS.value, Role.PUBLIC_USER.value])
+    def get(cls, org_id):
+        """Return all affiliation invitations associated with this org."""
+        status = request.args.get('status', None)
+        statuses = request.args.get('statuses', None)
+        filter_by_status = None
+
+        if status is not None:
+            filter_by_status = [status]
+        elif statuses is not None:
+            filter_by_status = statuses.split(',')
+
+        try:
+            cls._verify_permissions(org_id)
+
+            data = AffiliationInvitationService. \
+                get_all_invitations_with_details_related_to_org(org_id=org_id, statuses=filter_by_status)
+            data = [AffiliationInvitationService.as_dict_with_details
+                    (elem[0], elem[1], elem[2], elem[3]) for elem in data]
+
+            response, status = {'affiliationInvitations': data}, http_status.HTTP_200_OK
```

Comment:
This looks similar to one of the routes Odysseus built

Reply:
cleaned up code a little bit, though route might be similar I think they are doing different things in the service calls.

Reply:
So why not change the route he built? so we don't have two different versions that manage the same entity?